packages/kbn-generate-csv/src/generate_csv.ts

```diff
@@ -235,6 +237,21 @@ export class CsvGenerator {
 
   public async generateData(): Promise<TaskRunResult> {
     const logger = this.logger;
+
+    const createSearchSource = async () => {
+      try {
+        const source = await this.dependencies.searchSourceStart.create(this.job.searchSource);
+        return source;
+      } catch (err) {
+        // Saved object not found
+        if (err?.output?.statusCode === 404) {
+          const reportingError = new ReportingSavedObjectNotFoundError(err);
+          throw createTaskRunError(reportingError, TaskErrorSource.USER);
+        }
+        throw new Error(err);
```

Comment:
I think it should be

```suggestion
        throw err;
```

Reply:
Oops. That's right. I'll change this 👍🏻